go/slack/slack_test.go

```diff
@@ -0,0 +1,24 @@
+package slack
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+const (
+	configPath     = "../../config/config.yaml"
+	reportFilePath = "../../report/sample/sample_oltp.json"
```

Comment:
Not a deal-breaker, but perhaps these files should be auto-generated for test purposes. The paths assigned to these variables assume we are in the `./go/slack` directory.